.github/workflows/ci.yml

```diff
@@ -28,14 +28,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: install linux deps
-        run: |
-          sudo apt update
```

Comment:
The action's Ubuntu image comes with the package list already. No need to update unless we need a package that was first published several days ago.